internal/provisioner/kops_provisioner_cluster.go

```diff
@@ -26,9 +26,13 @@ import (
 	"github.com/mattermost/mattermost-cloud/model"
 )
 
-// DefaultKubernetesVersion is the default value for a kubernetes cluster
-// version value.
-const DefaultKubernetesVersion = "0.0.0"
+const (
+	// DefaultKubernetesVersion is the default value for a kubernetes cluster
+	// version value.
+	DefaultKubernetesVersion = "0.0.0"
+	// CustomNodePolicyName is the name of the custom IAM policy that will be attached in Kops Instance Profile
+	CustomNodePolicyName = "cloud-provisioning-node-policy"
```

Comment:
@gabrieljackson let me know if you think this makes more sense to add it in the /tools/aws/constants.go file.

Reply:
I could see it making sense in either place, but it may fit a bit better there, so yeah I would say we might as well move it.